.travis.yml

```diff
@@ -1,45 +1,77 @@
 language: node_js
-
+dist: trusty
 compiler: gcc
 sudo: false
 
 env:
   global:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
+    - NVS_VERSION=1.3.0
     - SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true
 
 jobs:
   include:
     - stage: test
-      node_js: "node"
+      env:
+        global:
+          - NODEJS_VERSION: node/6
```

Comment:
By default the `node` is the latest stable, currently 8.1.2. We'd want the equivalent to `node/latest`